app/presenters/ohms_transcript_display.rb

```diff
@@ -43,6 +43,27 @@ def display
     content_tag("div", safe_join(paragraph_html_arr), class: "ohms-transcript-container")
   end
 
+  def test_render
+    render "works/ohms_footnote_reference"
+  end
```

Comment:
is this method supposed to be called `test_render`?

Reply:
That was a typo left over from testing. Got rid of it.